Backend/autozon/.env.example

```diff
@@ -1,6 +1,6 @@
 APP_NAME=Laravel
```

Comment:
Don't push changes in .env or .env.example file (Keys shouldn't be on repository)